README.md

```diff
@@ -7,35 +7,41 @@ Printing since 1440.
+- **Download:** If you want to use the latest release with your WordPress, <a href="https://wordpress.org/plugins/gutenberg/">download the latest release from the WordPress.org plugins repository</a>. 
+- **Discuss:** Conversations and discussions take place in #core-editor on <a href="https://make.wordpress.org/chat/">the core WordPress Slack</a>.
+- **Contribute:** Development happens here. Get started by reading the <a href="https://github.com/WordPress/gutenberg/blob/master/CONTRIBUTING.md">CONTRIBUTING</a> guidelines.
+- **Learn:** Discover more about the project <a href="https://wordpress.org/gutenberg">here</a>.
 
-Gutenberg is more than an editor. While the editor is the focus right now, the project will ultimately impact the entire publishing experience including customization (the next focus area).
+**Gutenberg is more than an editor.** While the project is currently focused on building the new editor for WordPress, it doesn't end there. This lays the groundwork for a new model for WordPress Core that will ultimately impact the entire publishing experience of the platform.
 
 ## Editing focus
 
-> The editor will create a new page- and post-building experience that makes writing rich posts effortless, and has “blocks” to make it easy what today might take shortcodes, custom HTML, or “mystery meat” embed discovery. — Matt Mullenweg
+> *The editor will create a new page- and post-building experience that makes writing rich posts effortless, and has “blocks” to make it easy what today might take shortcodes, custom HTML, or “mystery meat” embed discovery.*
+>
+> — Matt Mullenweg
 
-One thing that sets WordPress apart from other systems is that it allows you to create as rich a post layout as you can imagine -- but only if you know HTML and CSS and build your own custom theme. By thinking of the editor as a tool to let you write rich posts and create beautiful layouts, we can transform WordPress into something users _love_, as opposed something they pick it because it's what everyone else uses.
+One thing that sets WordPress apart from other systems is that it allows you to create a post layout as rich as you can imagine—but only if you know HTML and CSS and build your own custom theme. By thinking of the editor as a tool that can let you both write rich posts and create beautiful layouts, we can transform WordPress into something users _love_, as opposed something they pick because it happens to be what everyone else uses.
 
-Gutenberg looks at the editor as more than a content field, revisiting a layout that has been largely unchanged for almost a decade. This allows us to holistically design a modern editing experience and build a foundation for things to come.
+**Gutenberg is a new way forward.** It looks at the editor as more than a content field, revisiting a layout that has been largely unchanged for almost a decade. This project allows us to holistically design a modern editing experience, and build a foundation for things to come.
```

Comment:
I've seen a bit of discussion recently about using 'us' or 'we', which doesn't read very inclusively. It could be something to change here.

Using 'the WordPress community' as a replacement might be an idea.

Reply:
Looks great! I have a minor quibble with the revision in that stylistically, there isn't a need for a comma after "experience" as the word "and" acts as a natural pause.